x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/TokenService.java

```diff
@@ -1910,6 +1919,30 @@ boolean isExpirationInProgress() {
         return expiredTokenRemover.isExpirationInProgress();
     }
 
+    public final class CreateTokenResult {
```

Comment:
This class can be `static` since it does not need need access any instance variables.

Reply:
Done